modules/loop_search_node/loop_search_node.features.inc

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * @file
+ * loop_search_node.features.inc
+ */
+
+/**
+ * Implements hook_ctools_plugin_api().
+ */
+function loop_search_node_ctools_plugin_api($module = NULL, $api = NULL) {
+  if ($module == "page_manager" && $api == "pages_default") {
+    return array("version" => "1");
+  }
+}
+
+/**
+ * Implements hook_default_search_api_index().
+ */
+function loop_search_node_default_search_api_index() {
+  $items = array();
```

Comment:
I denne funktion angives søge-API-indekset som et stort, konstant array. Eftersom nogle af disse indstillinger kan være noget, man kan være interesseret i at skrue på (for eksempel listen af stopwords på engelsk), hører de måske snarere hjemme i databasen eller en konfigurationsfil?